roles/mesos/defaults/main.yml

```diff
@@ -25,8 +25,8 @@ mesos_attributes:
 mesos_cluster: mantl
 
 mesos_zk_auth: "{% if zk_mesos_user_secret is defined %}{{ zk_mesos_user }}:{{ zk_mesos_user_secret }}@{% endif %}"
-mesos_zk_dns: "zookeeper.service.consul"
-mesos_zk_hosts: "{% for host in groups[zookeeper_server_group] %}{{ host }}:{{mesos_zk_port}}{% if not loop.last %},{% endif %}{% endfor %}"
+mesos_zk_dns: "zoiokeeper.service.consul"
```

Comment:
@siddharthist and @ChrisAubuchon  zoios are a new kind of zoo actually, haha will fix.